server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/BlobTables.java

```diff
@@ -48,4 +48,11 @@
         String CHUNK_NUMBER = "chunkNumber";
         String DATA = "data";
     }
+
+    interface DumbBlobCache {
+        String TABLE_NAME = "blobCache";
+        String ID = "id";
+        String DATA = "data";
+        String TTL = "ttl";
```

Comment:
what is this `TTL` for?

Reply:
time to live for evict cache

Reply:
bindMarker naming

Reply:
we usually have column names here, could we prefix it to make it more obvious it's not a column?

Reply:
Yes, any suggestion for it could be great.

Reply:
TTL_FOR_ROW